source/portable/NetworkInterface/xilinx_ultrascale/x_emacpsif_physpeed.c

```diff
@@ -842,6 +842,9 @@ static void ar8035Tick( XEmacPs * xemacpsp,
     value = XEmacPs_PhyRead2( xemacpsp, phy_addr, IEEE_STATUS_REG_OFFSET );
```

Comment:
`ar8035Tick`, `prvAR803x_debug_reg_mask`, `prvAR803x_debug_reg_read` : if they're unused and static, I wouldn't mind if they are removed.